docs/sdk.rst

```diff
@@ -329,3 +329,37 @@ execution time.
 
 .. |dill| replace:: ``dill``
 .. _dill: https://dill.readthedocs.io/en/latest/#basic-usage
+
+Avoiding Serialization Errors
```

Comment:
Let's move this docs addition to it's own PR.

Reply:
Pushed to the wrong branch last night with the other PR that isn't even ready.  Removing.

Reply:
Somehow this got pushed to the wrong PR, removing.